axelrod/strategies/worse_and_worse.py

```diff
@@ -12,12 +13,11 @@ class WorseAndWorse(Player):
 
     Names:
         - Worse and Worse: [PRISON1998]_
-
     """
 
     name = 'Worse and Worse'
     classifier = {
-        'memory_depth': float('inf'),
```

Comment:
It needs to know the current turn so does that not equate to knowing how long the game has been? (Thus infinite memory).